### ISSUE TITLE
Support for QuickTime ProRes alpha channel

### ISSUE THREAD

**Issue by MikOfClassX** [Nevcairiel/LAVFilters#90]
Hello,

I tested a QuickTime ProRes 4444 animation with alpha.
The LAV filter works very well, but it ignores the alpha channel which results completely filled.
The QuickTime codec does the right job.

Thanks!

Mik

**Comment by Nevcairiel** [comment]
There are currently no plans to support video with alpha, sorry.

**Comment by tararot2** [comment]
Hi Nevcairiel.
Are you talking about QuickTime codec only or with any other codec?
In my computer I can play videos with alpha channel using lagarith lossless codec but i'm having problems using it in different computers.

**Comment by MikOfClassX** [comment]
QuickTime video codecs are generally well supported by LAV.
Alpha is always supported, when present, except with the ProRes4444 which is decoded by masking the alpha channel as 0xff.
I don't think it would be so difficult to add alpha channel support here.

**Comment by richardpl** [comment]
Do you have samples?

**Comment by MikOfClassX** [comment]
Yes I have a sample .mov prores444 + alpha. It's 155MB. I can send it by wetransfer (I need an email) or.. feel free to suggest a better way.

**Comment by richardpl** [comment]
send it to onemda@gmail.com

**Comment by MikOfClassX** [comment]
done, you'll receive a wetransfer file soon.

**Comment by richardpl** [comment]
I definitely see alpha with mpv.io player. Perhaps this is LAVFilters only issue?

**Comment by Nevcairiel** [comment]
LAV Video does not output any alpha information, and as mentioned several posts above, its not really planned to start doing that either.

**Comment by MikOfClassX** [comment]
I am quite sure of that. It's not a complex issue, as LAV is only ignoring the alpha, but it's decoding the MOV file without any issue.
